src/ui/libs/DatalensChartkit/ChartKit/components/Widget/components/Table/utils/misc.ts

```diff
@@ -356,6 +360,139 @@ function extractCellActionParams(args: {cell: TableCell; head?: TableHead}) {
     return {};
 }
 
+function getValuesMap(selectedRows: TableRow[]) {
+    return selectedRows.reduce<Record<string, number>>((acc, row) => {
+        if (!('cells' in row)) {
+            return acc;
+        }
+
+        row.cells.forEach((cell) => {
+            if (typeof cell !== 'object' || !('value' in cell)) {
+                return;
+            }
+
+            const stringifiedValue = String(cell.value);
```

Comment:
Is it really possible to take only the values of the strings here?
Perhaps you should use `getRowActionParams`?